practice-two/src/styles/layout/footer.css

```diff
@@ -65,25 +69,39 @@
 }
 
 .list-icons-social a .youtube {
+  margin-top: 2px;
   background-position: -10px -97px;
 }
 
 @media screen and (min-width: 1440px) {
-  .footer-menu {
-    padding: 50px 285px 50px 195px;
-    display: grid;
-    grid-template-columns: repeat(4, 152px) 321px;
+  .footer-menu .container,
+  .footer-social .container {
+    width: 1050px;
```

Comment:
**container** class should be style as common layout, shouldn't style for each section like that